.github/config.yml

```diff
@@ -13,18 +13,21 @@ newIssueWelcomeComment: |
 # Comment to be posted to on PRs from first time contributors in your repository
 newPRWelcomeComment: |
  Thanks for opening this pull request!  This space is [protected by our Code of Conduct](https://publiclab.org/conduct) - and we're here to help.
- Please make sure you've bumped up the version number in the repository's [package.json file](https://github.com/publiclab/PublicLab.Editor/blob/master/package.json) and that you've incorporated your changes in the [/dist/ sub-directory](https://github.com/publiclab/PublicLab.Editor/tree/master/dist).
+ Please make sure you've bumped up the version number in the repository's [package.json file](https://github.com/publiclab/PublicLab.Editor/blob/main/package.json) and that you've incorporated your changes in the [/dist/ sub-directory](https://github.com/publiclab/PublicLab.Editor/tree/main/dist) by running `grunt build`.
  `Dangerbot` will test out your code and reply in a bit with some pointers and requests.
  Also please refer [here](https://github.com/publiclab/PublicLab.Editor#installation) for installation help 💿    
- There may be some errors, **but don't worry!** We'll work through them with you! 👍🎉😄    
- It would be great if you can tell us your Twitter handle so we can thank you properly?
+ There may be some errors, **but don't worry!** We'll work through them with you! 👍🎉😄
+ <br />
+ One thing that can help to get started is to make sure you've included a link back to the original issue you're solving, in the format `fixes #0000` (for example). And to make sure the PR title describes what you're trying to do! (often it can be the same as the issue title) Thanks! 🙌
+ <br />
+ Then, you can [say hello in our chatroom](https://gitter.im/publiclab/publiclab) & share a link to this PR to get a review! 👋 ✅
 # Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
 
 # Comment to be posted to on pull requests merged by a first time user
 firstPRMergeComment: |
  Congrats on merging your first pull request! 🙌🎉⚡️ 
- Your code will likely be published to PublicLab.org in the next few days. In the meantime, can you tell us your Twitter handle so we can thank you properly?
- Do join our [weekly check-in](https://github.com/publiclab/plots2/issues/5627) to share your this week goal and the awesome work you did 😃.
+ Your code will likely be published to PublicLab.org in the next few days.
+ Do join our [weekly check-in](https://github.com/publiclab/plots2/issues/9600) to share your this week goal and the awesome work you did 😃.
```

Comment:
Hi @taylorkaiulani, I know this isn't in the original issue but would you mind changing `share your this week goal` to `share your goal for this week`. Thanks and sorry for the inconvenience!

Reply:
Will do!

Reply:
Okay, I committed the change. Let me know if there's anything else I need to do!